app/Http/Controllers/MappingController.php

```diff
@@ -39,24 +40,39 @@ public function create(Webhook $webhook)
      * @param  \Illuminate\Http\Request  $request, Webhook $webhook
      * @return \Illuminate\Http\Response
      */
-    public function store(MappingCreateRequest $request, Webhook $webhook)
+    public function store(MappingCreateRequest $request, $webhookId)
     {
+        $webhook = Webhook::findOrFail($webhookId);
         $this->authorize('create', [new Mapping(), $webhook]);
 
-        $attributes = $request->except('_token');
-        $attributes['webhook_id'] = $webhook->id;
-
+        DB::beginTransaction();
         try {
-            $mapping = $this->mappingRepository->create($attributes);
+            $keys = $request->keys;
+            $values = $request->values;
+            if ($keys && $values) {
+                for ($i = 0; $i < count($keys); $i++) {
+                    $attributes['webhook_id'] = $webhookId;
+                    $attributes['key'] = $keys[$i];
+                    $attributes['value'] = $values[$i];
+                    $mapping = Mapping::where('webhook_id', $webhookId)->updateOrCreate(['key' => $attributes['key']], $attributes);
+                }
+            }
+
+            DB::commit();
+            $request->session()->flash('messageSuccess', [
+                'status' => 'Create success',
+                'message' => 'This mapping successfully created',
+            ]);
 
-            return redirect()->route('webhooks.mappings.edit', ['webhook' => $webhook, 'mapping' => $mapping])
-                ->with('messageSuccess', [
-                    'status' => 'Create success',
-                    'message' => 'This mapping successfully created',
-                ]);
-        } catch (QueryException $e) {
-            return back()->with('messageFail', [
-                'status' => 'Create failed',
+            return response()->json([
+                'error' => false,
+                'webhook_id' => $webhook->id
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/MappingController.php</br>**Line**: 69</br>Each line in an array declaration must end in a comma